Makefile

```diff
@@ -5,35 +5,32 @@ GOLINT = golint
 GOTEST = $(GO) test --cover --race -v
 GOVET = $(GO) vet
 GO_FILES = $(wildcard *.go)
-GO_PACKAGES = storage syslogish weblog
+GO_PACKAGES = syslogish
 GO_PACKAGES_REPO_PATH = $(addprefix $(REPO_PATH)/,$(GO_PACKAGES))
 GO_TESTABLE_PACKAGES_REPO_PATH = $(addprefix $(REPO_PATH)/,storage storage/file storage/ringbuffer)
 
 # the filepath to this repository, relative to $GOPATH/src
-REPO_PATH = github.com/deis/nginxpusher
+REPO_PATH = github.com/deis/stdout-metrics
 
 # The following variables describe the containerized development environment
 # and other build options
-DEV_ENV_IMAGE := quay.io/deis/go-dev:0.7.0
+DEV_ENV_IMAGE := quay.io/deis/go-dev:0.11.0
```

Comment:
This could be updated to `go-dev:0.11.2` to get Go 1.6.2. (Also strictly speaking this should be a separate commit.)